server.js

```diff
@@ -112,7 +115,14 @@ const { ResourcePagesRouter } = require("./newroutes/resourcePages")
 const { ResourceRoomRouter } = require("./newroutes/resourceRoom")
 const { SettingsRouter } = require("./newroutes/settings")
 const { UnlinkedPagesRouter } = require("./newroutes/unlinkedPages")
+const {
+  AuthMiddlewareService,
+} = require("./services/middlewareServices/authMiddlewareService")
+
+const authMiddlewareService = new AuthMiddlewareService()
+const authMiddleware = new AuthMiddleware({ authMiddlewareService })
 
+const authService = new AuthService()
```

Comment:
._. this is p confusing and we could have a helper called `initAuthMiddleware` that just does this

Reply:
As discussed, this is a good point that warrants further team discussion - but will be out of scope of this PR regardless